client/src/Components/Mentors.js

```diff
@@ -137,31 +137,31 @@ export default function Mentors(props) {
 	};
 	if (filteredQuestionsByTag) {
 		return (
-			<div className='main'>
+			<div>
 				<Navbar mentors="Mentors" results="Results" newquestion ="New Question" />
 				<div className="container">
 					<div className='row'>
-					{modalText?(modalText==="OK"?<Modal modalText={"submitted successfully"} />:<Modal modalText={"something went wrong"} />):null}
+						{modalText?(modalText==="OK"?<Modal modalText={"submitted successfully"} />:<Modal modalText={"something went wrong"} />):null}
 						<RunQuiz quizzes={props.quizzes} />
 						<div className="filterButtons row">
-						<div>
-					 <select class="btn btn-light dropdown-toggle pt-2" onChange={selectHandler}>
+							<div>
+					 <select className="btn btn-light dropdown-toggle pt-2" onChange={selectHandler}>
 						 <option selected disabled>Number of question</option>
 									<option value="5">5</option>
 									<option value="10">10</option>
 									<option value="15">15</option>
 									<option value="20">20</option>
 								</select>
 					 </div>
-							<select class="btn btn-light dropdown-toggle ml-2" onChange={tagClickHandler}>
+							<select className="btn btn-light dropdown-toggle ml-2" onChange={tagClickHandler}>
```

Comment:
should be fine. please check for conflicts because i merged not long ago some of this changes as well